federation/cmd/federation-apiserver/app/server.go

```diff
@@ -48,8 +48,8 @@ import (
 	"k8s.io/kubernetes/pkg/generated/openapi"
 	"k8s.io/kubernetes/pkg/kubeapiserver"
 	kubeapiserveradmission "k8s.io/kubernetes/pkg/kubeapiserver/admission"
-	kubeoptions "k8s.io/kubernetes/pkg/kubeapiserver/options"
-	kubeserver "k8s.io/kubernetes/pkg/kubeapiserver/server"
```

Comment:
It may not be very well named (we were struggling for names at the time), but `k8s.io/kuberentes/pkg/kubeapiserver` is actually a package shared between the kube-apiserver and federation.